src/aoai-api-simulator/src/aoai_api_simulator/config_loader.py

```diff
@@ -51,9 +52,18 @@ def _load_openai_deployments(logger: logging.Logger) -> dict[str, OpenAIDeployme
         config_json = json.load(f)
     deployments = {}
     for deployment_name, deployment in config_json.items():
+        model_name = deployment["model"]
+
+        if model_name not in model_catalogue:
+            logger.error(
```

Comment:
Could you also update this line to use the `%s` substitution? Sorry, didn't spot this before